solidus_virtual_gift_card.gemspec

```diff
@@ -17,7 +17,7 @@ Gem::Specification.new do |s|
   s.require_path = "lib"
   s.requirements << "none"
 
-  s.add_dependency "solidus", "~> 1.2"
+  s.add_dependency "solidus_core", "~> 1.2"
```

Comment:
This is not a safe dependency change. There are also dependencies on the API and backend in controller inheritance.